float32 number1
float32 number2
---
float32 result
